myapp/spec/models/article_spec.rb

```diff
@@ -0,0 +1,19 @@
+require 'rails_helper'
+
+RSpec.describe Article, type: :model do
+  it "has a valid factory" do
+    expect(FactoryGirl.create(:article)).to be_valid
+  end
+
+  it "without title is invalid" do
```

Comment:
Here you could also add that article.errors['title'] is not empty for example. But this is okay